src/app/Caches/Create/BasicCacheConfigConfigurator.tsx

```diff
@@ -61,6 +61,22 @@ const BasicCacheConfigConfigurator = () => {
           maxIdleNumber: maxIdleNumber,
           maxIdleUnit: maxIdleUnit,
           valid: lifeSpanNumber >= -1 && maxIdleNumber >= -1
+        },
+        feature: {
+          ...prevState.feature,
+          indexedCache: {
+            ...prevState.feature.indexedCache,
+            valid: configuration.feature.cacheFeatureSelected.includes(CacheFeature.INDEXED)
```

Comment:
extract this to methods. maybe we can create a helper class that will test these methods with jest in test unit to avoid too much integration test